### PR TITLE
[CI] Fix docs deploy & links

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: .woodpecker/docs.yml [hunk 1]
```diff
@@ -21,10 +21,10 @@ pipeline:
       - chmod 0600 $HOME/.ssh/id_rsa
       - cd docs/
       - yarn install --frozen-lockfile
+      - yarn build
       - yarn deploy
     when:
-      event: push
-      branch: master
+      event: [pull_request]
```
Comment (anbraten) [inline .woodpecker/docs.yml:27]: Are you sure we should deploy on PRs? This was initially the job to update the docs on pushes to master. `yarn deploy` should build and deploy the docs to our github page.

**Review by anbraten — CHANGES_REQUESTED** [review]

File: .woodpecker/docs.yml (hunk 1 above)
Comment (6543) [reply .woodpecker/docs.yml:27]: no it's for testing porpose - but didn't worked out

**Comment by anbraten** [comment]
Seems to almost work now, but I can't find the new files from your test run on `woodpecker-ci/woodpecker-ci.github.io` 🤔 Although the CI steps says it finished correctly.

```
CMD: git commit -m "Deploy website - based on 2aa5adeb34e59a3b01b2ce5f14c446faaffe7b29" (code: 0)
To github.com:woodpecker-ci/woodpecker-ci.github.io.git
   00ac224..4aecf06  master -> master
CMD: git push --force origin master (code: 0)
Website is live at "https://woodpecker-ci.github.io/".
Done in 59.08s.
```

**Review by anbraten — APPROVED** [review]
Ahh the docs repo got reset. So fine from my side.

**Review by anbraten — APPROVED** [review]
lgtm

**Pull request merged**